[OSDOCS#17058]: DC-2: Image Mirroring and oc mirror Tooling-CQA-2 (#115637)
openshift/openshift-docs: modules/mirror-registry-remote-host-update.adoc

[id="mirror-registry-remote-host-update_{context}"]
= Updating mirror registry for Red Hat OpenShift from a remote host

[role="_abstract"]

> This procedure explains how to update the _mirror registry for Red{nbsp}Hat OpenShift_ from a remote host by using the `upgrade` command. Updating to the latest version ensures bug fixes and security vulnerability fixes.

Reviewer: ```suggestion
You can update the _mirror registry for Red{nbsp}Hat OpenShift_ from a remote host by using the `upgrade` command. Updating to the latest version ensures bug fixes and security vulnerability fixes.
```

I will stop repeating this comment now, please apply similar feedback to all procedure abstracts that start with "This procedure..."
